### DISCUSSION TITLE
Remove extra cover page when using user-defined cover

### DISCUSSION THREAD

**Discussion by sonarventu** [joaomlourenco/novathesis#447] [Q&A] [answered]
Hello,
I'm doing my PhD in an Italian university, and so I personalized the front cover of the Thesis by inserting my own, following a template provided by my institution.
Still, the template is printing a second cover -- i.e., the one showing the committee and similar information. Since including this information is not required in my institution, is there a way not to print this second page at all?
Thanks a lot!
Davide

**Comment by joaomlourenco** [comment]
Yes. There is an option in `Config/1_novathesis.tex` to force the removal of the second cover.
```latex
95:% \ntsetup{print/secondcover=true}    % Shall a second cover page be forced?
96:%                               %     if the contents for the second page are not
97:%                               %     defined, the second cover will be a replica
99:%                               % DEFAULT: print/secondcover=false
```

Probably the University /School you used as basis for your adaptation is forcing it.  Try changing the above first… it it does not work, search for a similar command in your “university_schoool.tex” file.

**Reply by sonarventu** [reply]
Actually I tried to set `print/secondcover=false` anywhere I could find it, but it still gets printed...
I don't know if it's relevant, but to insert my own cover I went to files.tex, and under `User defined cover instead of standard one ` I activated `\ntaddfile{cover}[1]{cover-front} `, where cover-front.tex is my cover file.

**Reply by sonarventu** [reply]
@joaomlourenco actually I made some progress on the problem. The fact is, what I'm seeing is not the second cover (as I initially thought), but actually the first. The problem is, when I included my own cover using `User defined cover instead of standard one`, the template still prints the default cover after printing mine.

So my question becomes: is there a way not to print the _first_ and default cover, since I have already included my user-defined one?
Thanks a lot!

**Reply by sonarventu** [reply, marked as answer]
I temporarily fixed it by explicitly commenting out all the instructions following "COVER 2" in `nova-fct-defaults.ldf`.
So I guess this is actually a bug, in the sense that `print/secondcover=false` is not working properly in the presence of a user-defined cover.

**Comment by joaomlourenco** [comment]
May I ask you to share your adaptation of the cover to your university?

**Reply by sonarventu** [reply]
Sure, but it frankly looks very bad :) it can be found [here](https://math.sissa.it/content/mama-phd-cover)
In fact, I did not adapt the existing cover file, but simply replaced it with the one given by my university.

**Comment by sonarventu** [comment]
@joaomlourenco I'm very sorry to insist, but I have to hand in my thesis in a week and I still haven't found a solution to this problem... Is there really nothing I can do? Thanks a lot...

**Reply by joaomlourenco** [reply]
Hi. The answer is yes… of course I'll help you.  :)
Can you provide me access to your repository (github? overleaf?)
Or, in alternative, give me a link to download a zip?
You may remove all the chapters but one (e..g., the first one).
Mail me at "joao.lourenco [AT] fct [DOT] unl [DOT] pt".

**Reply by sonarventu** [reply]
Hi @joaomlourenco, thanks a lot for your kind offer! I don't want to take up even more of your time unnecessarily, though: for what concerns my thesis, the problem has been virtually solved by removing those extra lines in `nova-fct-defaults.ldf` (see my last comment above).

Of course I understand that such a solution is not elegant. Maybe I messed something up and it was ultimately my fault. Still, I suggest to try to see if the bug persists in a minimal working example, where a user-defined cover (i.e., any single-page tex file) is inserted... What do you think? 

Davide

**Reply by joaomlourenco** [reply]
Hi Davide,
I'm happy that in the end it worked for you.
I'll double check the effectiveness of `print/secondcover=false` with a minimal example.
Thanks and good luck to your defense.
